tc/core/polyhedral/llvm_jit.cc

```diff
@@ -71,42 +71,6 @@ std::string find_library_path(std::string library) {
 
 namespace tc {
 
-#if LLVM_VERSION_MAJOR <= 6
```

Comment:
Does this mean we now require LLVM 7?

Reply:
yes, we are using an LLVM close to trunk since 7dbb65b622d03e5d3a36578e81e781f1c9ad8d6c